### PR TITLE
Add __('<your_string>') method in Button Example

### PR DESCRIPTION
**Pull request by hardikm-cirkleinc**
## Purpose of this pull request

For Modal widget, Add __('<your_string>') method in Button Example because this method is used for translation of the string.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_modal.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_modal.html
- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_modal.html

## Links to Magento source code

-https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/modal/modal.js#L80
-https://github.com/magento/magento2/blob/2.2/app/code/Magento/Ui/view/base/web/js/modal/modal.js#L80
-https://github.com/magento/magento2/blob/2.1/app/code/Magento/Ui/view/base/web/js/modal/modal.js#L76

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4700) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4700) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@hardikm-cirkleinc thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by hardikm-cirkleinc** [comment]
I want to pull "**Add __('<your_string>') method in Button Example**" commits

File: guides/v2.1/javascript-dev-guide/widgets/widget_modal.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Array of buttons for action pane.
 **Structure**:
 ```javascript
 buttons: [{
-    text: '',
+    text: $.mage.__(''),
```
Comment (ronak2ram) [inline guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]:
```suggestion
    text: '',
```

Comment (hardikm-cirkleinc) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Yes this one

Comment (ronak2ram) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Hi @hardikm-cirkleinc 
Thanks for your contribution.
I think we don't need to do any change on line no 74.
Can you please revert change on line 74. we accept `$.mage.__('Ok')` this change.

Comment (hardikm-cirkleinc) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Hello @ronak2ram 
Thank you for your suggestion
I have changed line no 74 and I have also added this in line no 83.

Comment (ronak2ram) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Can you please revert change on line 74.

Comment (hardikm-cirkleinc) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Hello @ronak2ram 

I have reverted

Comment (hardikm-cirkleinc) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Can you Please approve the changes

Comment (ronak2ram) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: I will approve soon. Can you please cover changes of PR #4702 in current PR. so we can decrease duplicate efforts.

Comment (hardikm-cirkleinc) [reply guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Done bro

**Comment by hardikm-cirkleinc** [comment]
Hello, 

Can you please tell me when the pull changes ?

**Pull request merged**